precompiles/distribution/tx.go

```diff
@@ -93,7 +92,7 @@ func (p *Precompile) ClaimRewards(
 		if err != nil {
 			return nil, err
 		}
-		p.SetBalanceChangeEntries(cmn.NewBalanceChangeEntry(withdrawerHexAddr, totalCoins.AmountOf(utils.BaseDenom).BigInt(), cmn.Add))
+		p.SetBalanceChangeEntries(cmn.NewBalanceChangeEntry(withdrawerHexAddr, totalCoins.AmountOf(evmostypes.BaseDenom).BigInt(), cmn.Add))
```

Comment:
here we have to check if the total coins withdrawn from the rewards includes the evm denom. If there'r no coins with evm denom, we should not add the balance change entry. Only the evm denom amount should be added to the balance change entry.

Reply:
why don't we get the evm denom from the config here?